src/aldente.cpp

```diff
@@ -1,551 +1,99 @@
 #include "aldente.h"
+
 #include "window.h"
-#include "shaders/skybox_shader.h"
-#include "shaders/shadow_shader.h"
-#include "geometry/geometry_generator.h"
-#include "scene/scene_model.h"
-#include "scene/scene_transform.h"
-#include "scene/scene_animation.h"
-#include "bounding_sphere.h"
-#include <cfloat>
-#include <string>
+#include "setup.h"
+#include "asset_loader.h"
+#include "physics.h"
+#include "shaders/shader_manager.h"
+#include "shadows.h"
+
+#include "keyboard.h"
 
 #include "events/input.h"
 #include "input/process.h"
 #include "util/util.h"
-#include "util/colors.h"
-#include "global.h"
-#include "util/config.h"
-#include "btBulletDynamicsCommon.h"
-
-#include <glm/gtc/matrix_transform.hpp>
-#include <glm/gtc/type_ptr.hpp>
 
 namespace input = kuuhaku::input;
 namespace events = kuuhaku::events;
 
-/* global vars */
-Scene* scene;
-std::vector<Scene*> scenes;
-SceneCamera* camera;
-
-bool keys[1024];
-bool lmb_down = false;
-bool rmb_down = false;
-bool debug_shadows = false;
-bool mouse_moved = false;
-bool shadows_on = true;
-bool god_mode = false;
-bool helicopter_mode = false;
-glm::vec3 last_cursor_pos;
-
-const GLfloat PLAYER_HEIGHT = Global::PLAYER_HEIGHT;
-
-const GLfloat FAR_PLANE = 50.f * PLAYER_HEIGHT;
-const GLfloat FOV = 45.f;
-
-const GLfloat   BASE_CAM_SPEED = PLAYER_HEIGHT / 10.f;
-const GLfloat   EDGE_PAN_THRESH = 5.f;
-const GLfloat   EDGE_PAN_SPEED = 0.5f;
-
-//Bullet varaibles
-btBroadphaseInterface* broadphase;
-btDefaultCollisionConfiguration* collisionConfiguration;
-btCollisionDispatcher* dispatcher;
-btSequentialImpulseConstraintSolver* solver;
-btDiscreteDynamicsWorld* dynamicsWorld;
-vector<btRigidBody*> rigidBodies;
-
-Grid* grid;
-Tile* hover;
+Aldente* Aldente::aldente = new Aldente();
 
 Aldente::Aldente() {}
-
 Aldente::~Aldente() {}
```

Comment:
Can remove constructor and destructor since they're just their respective defaults